app/models/mixins/retirement_mixin.rb

```diff
@@ -198,6 +198,10 @@ def error_retiring?
 
   private
 
+  def retire_queue_options
+    try(:my_zone) ? {:zone => my_zone} : {}
```

Comment:
@tinaafitz If `my_zone` returns `nil` this will pass `{:zone => nil}`.  Will that have the expected results?  Or should we only pass the zone_name if we have an actual name?  It is unclear to me which is correct.

cc @mkanoor